### ISSUE TITLE
Error: Only 65535 characters are allowed

### ISSUE THREAD

**Issue by MrGadget1024** [game-ci/unity-test-runner#214]
Still getting this error, not fixed by https://github.com/game-ci/unity-test-runner/commit/b9980d6db77b429efa66b950aef387eea018775d

```yml
- name: Run editor Tests
  continue-on-error: true
  uses: game-ci/unity-test-runner@main
```

```
Processing file editmode-results.xml...
Trying to open artifacts/editmode-results.xml
File artifacts/editmode-results.xml parsed...
Start analyzing results: editmode-results.xml
⚠️ editmode-results.xml - 991/1003, skipped: 12 - Passed in 107.210s
=================
Analyze result:
⚠️ Test Results - 991/1003, skipped: 12 - Passed in 107.210s
Posting results for 953944bREDACTEDf5fc69cf8e7c906d266cef5ef
Error: Invalid request.

Only 65535 characters are allowed; 75929 were supplied.
```

**Comment by MrGadget1024** [comment]
Any progress on this, or ideas?

**Comment by timcassell** [comment]
I'm surprised #206 didn't fix it. You can work around it by using another test reporting tool (see comments in #142).

**Comment by MrGadget1024** [comment]
> I'm surprised #206 didn't fix it. You can work around it by using another test reporting tool (see comments in #142).

Are we talking about the same point of failure?  The failure I'm getting is before the archive step...

![image](https://user-images.githubusercontent.com/9826063/222278548-5ad68d58-9927-48d5-b514-1e0b4172836e.png)

**Comment by timcassell** [comment]
Yes, that looks like the same error to me. I see you are using another test reporter. Remove `githubToken: ${{ secrets.GITHUB_TOKEN }}` to disable the built-in test reporter.

**Comment by MrGadget1024** [comment]
> Yes, that looks like the same error to me. I see you are using another test reporter. Remove `githubToken: ${{ secrets.GITHUB_TOKEN }}` to disable the built-in test reporter.

This got rid of the error but leaving this ticket open as the issue still exists.  Will work on a nicer test reporter tomorrow.

**Comment by johanhelsing-attensi** [comment]
Also hitting this one. Let me know if there's anything I can do to help.

**Comment by webbertakken** [comment]
I think some worked around this by parallelising tests to different jobs. This might not be a solution for everyone though.

---

One idea we've had is to [group](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines) logs on a per test suite basis.

We could most likely achieve that by optionally writing the unity logs to a file instead of `stdout`, and then doing some regex to specify the delimiters.

**Comment by MrGadget1024** [comment]
@davidmfinol Should this be closed now?

**Comment by davidmfinol** [comment]
@MrGadget1024 can you confirm it works OK now if you use `game-ci/unity-test-runner@main`?

**Comment by MrGadget1024** [comment]
@davidmfinol Someone else like @johanhelsing-attensi will have to check...My workflow has changed and I wouldn't be able to reproduce it now.

**Comment by migwellian** [comment]
Hello, I can confirm it is fixed - I had the same issue so I switched from `game-ci/unity-test-runner@v4` to `game-ci/unity-test-runner@main` and it fixed the issue.

**Comment by davidmfinol** [comment]
Fix released in v4.3.1
